packages/fluentui/react-builder/src/components/ReaderNarration.tsx

```diff
@@ -16,98 +17,119 @@ export type ReaderNarrationProps = {
 
 export const ReaderNarration: React.FunctionComponent<ReaderNarrationProps> = ({ selector, inUseMode }) => {
   const ref = React.useRef<HTMLElement>();
-  const [selectedNarrationPath, setSelectedNarrationPath] = React.useState(null);
+  const [narrationElement, setNarrationElement] = React.useState(null);
```

Comment:
Is there a type you can assign here ? otherwise it's a very confusing implicit any

```suggestion
  const [narrationElement, setNarrationElement] = React.useState<type>(null);
```

Reply:
Fixed to `React.useState<IAriaElement>`.